Automattic/wp-calypso
# Full Site Editing Design Picker: Enable Russell theme

#### Changes proposed in this Pull Request

* Update `available-designs-config.json` to include new Russell block theme.
* Russell is a selectable theme in the `/new` FSE beta onboarding design picker

#### Screenshots
<img width="1282" alt="Screen Shot 2021-11-30 at 5 20 56 PM" src="https://user-images.githubusercontent.com/5414230/144154576-690de611-0c62-4b47-91dd-1c185e1eb34d.png">

#### Testing instructions



* Apply branch to local calypso dev environment
* Visit http://calypso.localhost:3000/new/beta
* Enroll in the FSE beta
* Pick a free test domain
* Verify that Russell is a selectable theme
* Create a site with Russell enabled and smoke test the theme



Related to # https://github.com/Automattic/wp-calypso/issues/58663
